lib/helpers.js

```diff
@@ -123,7 +146,9 @@ module.exports = {
    */
   log(msg) {
     /* istanbul ignore next */
-    if (!global.isSilent) console.log(msg)
+    if (global.isSilent) return
+    if (global.isStdout) console.error(msg)
```

Comment:
I think we should all the message into `stderr`, but our previous implementation was using `stdout` to show the message. So this line send the error to `stderr` only when we use `stdout` for compiled output.

- **riot**
  - `stdout`: error messages via `console.log()`
- **riot --stdout**
  - `stdout`: compiled outputs via `process.stdout.write()`
  - `stderr`: error messages via `console.error()`